pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -3,22 +3,22 @@
     "Microsoft.CodeAnalysis.CSharp": "1.3.0",
     "Microsoft.CodeAnalysis.VisualBasic": "1.3.0",
```

Comment:
Agreed. I wanted to minimize the already large delta for this PR but eventually these should all be bumped to latest.